src/js/_enqueues/wp/updates.js

```diff
@@ -2462,4 +2471,151 @@
+				$parent = $anchor.closest( '.theme-autoupdate' );
+			}
+
+			event.preventDefault();
+
+			// Prevent multiple simultaneous requests.
+			if ( $anchor.attr( 'data-doing-ajax' ) === 'yes' ) {
+				return;
+			}
+
+			$anchor.attr( 'data-doing-ajax', 'yes' );
+
+			switch ( pagenow ) {
+				case 'plugins':
+				case 'plugins-network':
+					type = 'plugin';
+					asset = $anchor.closest( 'tr' ).attr( 'data-plugin' );
+					break;
+				case 'themes-network':
+					type = 'theme';
+					asset = $anchor.closest( 'tr' ).attr( 'data-slug' );
+					break;
+				case 'themes':
+					type = 'theme';
+					asset = $anchor.attr( 'data-slug' );
+					break;
+			}
+
+			// Clear any previous errors.
+			$parent.find( '.notice.error' ).addClass( 'hidden' );
+
+			// Show loading status.
+			// TODO: make it readable when network is fast, or possibly remove the interim text change.
+			if ( 'enable' === action ) {
+				$label.text( wp.updates.l10n.autoUpdatesEnabling );
+			} else {
+				$label.text( wp.updates.l10n.autoUpdatesDisabling );
+			}
+
+			// TODO: Needs design review - the link text jumps under the mouse (part may get selected).
+			$anchor.find( '.dashicons-update' ).removeClass( 'hidden' );
+
+			data = {
+				action: 'toggle-auto-updates',
+				_ajax_nonce: settings.ajax_nonce,
+				state: action,
+				type: type,
+				asset: asset,
+			};
+
+			$.post( window.ajaxurl, data )
+				.done( function( response ) {
+					var $enabled, $disabled, enabledNumber, disabledNumber, errorMessage;
+					var href = $anchor.attr( 'href' );
+
+					if ( ! response.success ) {
+						// if WP returns 0 for response (which can happen in a few cases
+						// that aren't quite failures), output the general error message,
+						// since we won't have response.data.error.
+						if ( response.data && response.data.error ) {
+							errorMessage = response.data.error;
+						} else {
+							errorMessage = wp.updates.l10n.autoUpdatesError;
+						}
+
+						$parent.find( '.notice.error' ).removeClass( 'hidden' ).find( 'p' ).text( errorMessage );
+						wp.a11y.speak( errorMessage, 'polite' );
+						return;
+					}
+
+					// Update the counts in the enabled/disabled views if on a screen
+					// with a list table.
+					if ( 'themes' !== pagenow ) {
+						$enabled       = $( '.auto-update-enabled span' );
+						$disabled      = $( '.auto-update-disabled span' );
+						enabledNumber  = parseInt( $enabled.text().replace( /[^\d]+/g, '' ) ) || 0;
+						disabledNumber = parseInt( $disabled.text().replace( /[^\d]+/g, '' ) ) || 0;
+
+						switch ( action ) {
+							case 'enable':
+								++enabledNumber;
+								--disabledNumber;
+								break;
+							case 'disable':
+								--enabledNumber;
+								++disabledNumber;
+								break;
+						}
+
+						enabledNumber = Math.max( 0, enabledNumber );
+						disabledNumber = Math.max( 0, disabledNumber );
+
+						$enabled.text( '(' + enabledNumber + ')' );
+						$disabled.text( '(' + disabledNumber + ')' );
```

Comment:
This all would be much easier if the numbers are passed to the script via `wp_localize_script()` and then `wp.i18n._n()` is used.